politeiawww/config.go

```diff
@@ -70,6 +70,7 @@ const (
 	// User database options
 	userDBLevel     = "leveldb"
 	userDBCockroach = "cockroachdb"
+	userDBMysql     = "mysqldb"
```

Comment:
Use `MySQL`.  Its an acronym so should be capitalized and this is how the rest of the codebase references it so you should too.  Match the style of the codebase you're contributing to.